### PR TITLE
Initial README

### PR DESCRIPTION
**Pull request by orelmisan**
Add minimal configuration and execution instructions.

### CODE REVIEW

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/e3c6d9baea857904afc3bb7d8a9d6926885aa5c5..1fc4a55c8bca81c8fc282b7ece5a1d36bd8be8d3): Added pod exec permission after PR #43 was merged.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,136 @@
...
+| spec.param.NUMASocket                       | The NUMA node where the workloads shall be scheduled to                                                           | True         |                                                                                                        |
+| spec.param.networkAttachmentDefinitionName  | NetworkAttachmentDefinition name of the SR-IOV NICs connected                                                     | True         | Assumed to be in the same namespace                                                                    |
+| spec.param.trafficGeneratorRuntimeClassName | [Runtime Class](https://kubernetes.io/docs/concepts/containers/runtime-class/) the traffic generator pod will use | True         |                                                                                                        |
+| spec.param.trafficGeneratorImage            | Traffic generator's container image                                                                               | False        | Defaults to the U/S image https://quay.io/repository/kiagnose/kubevirt-dpdk-checkup-traffic-gen:latest |
```
Comment (RamLavi) [inline README.md:76]: what about dpdk image?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,136 @@
 # kubevirt-dpdk-checkup
+
 checkup validating DPDK readiness of cluster, using the Kiagnose engine
```
Comment (RamLavi) [inline README.md:3]: In order to set these the user needs to have the correct security permissions. Can you add something about that?

File: README.md (hunk 1 above)
Comment (orelmisan) [reply README.md:76]: It is part of PR #51 which is not yet merged.

**Review by RamLavi — APPROVED** [review]
/lgtm

**Pull request merged**